cve_bin_tool/extractor.py

```diff
@@ -179,3 +190,9 @@ class Extractor(BaseExtractor):
 
     def __call__(self, *args, **kwargs):
         return TempDirExtractorContext(*args, **kwargs)
+
+
+if __name__ == "__main__":
```

Comment:
This looks like some leftover debugging that we should probably remove.

Reply:
Yeah cleaning it.